apps/playground/src/app/components/forms/checkbox/checkbox.component.ts

```diff
@@ -79,4 +80,9 @@ export class CheckboxComponent implements OnInit {
   public toggleInlineHelp(): void {
     this.showInlineHelp = !this.showInlineHelp;
   }
+
+  protected onSubmitContactMethod(): void {
```

Comment:
I realize this is just the playground but I did expect the "agree to terms and conditions" checkbox to be marked as touched as well